src/App.js

```diff
@@ -9,7 +10,11 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <Article
+    title={article.title}
+    date={article.date}
+    text={article.text}
+  />
```

Comment:
```suggestion
  <Article {..article} />
```

or you can do it like this
https://medium.com/@kevinyckim33/jsx-spread-operator-component-props-meaning-3c9bcadd2493